sdk/src/types/block/output/nft.rs

```diff
@@ -79,6 +81,13 @@ impl NftOutputBuilder {
         self
     }
 
+    /// Sets the mana to the provided value. (∩ ͡° ͜ʖ ͡°)⊃━☆ﾟ. *
```

Comment:
Cute but no 😆

Reply:
That can be arranged

Reply:
Are you really insisting on keeping the emoji or can we remove it?